pkg/bundle/client.go

```diff
@@ -140,17 +141,27 @@ func (bc *bundleClient) GetActiveBundleNamespacedName(ctx context.Context) (type
 	return nn, nil
 }
 
-func (bc *bundleClient) GetBundleList(ctx context.Context) (bundles []api.PackageBundle, err error) {
+func (bc *bundleClient) GetBundleList(ctx context.Context, serverVersion string) (bundles []api.PackageBundle, err error) {
```

Comment:
It looks like this function could return bundles that don't match the current server version. Is that something we want to allow? Like for upgrades? Or, should this method also filter out any bundles that don't match the server version?